lib/stages/visAccumulate.cpp

```diff
@@ -63,6 +63,7 @@ visAccumulate::visAccumulate(Config& config, const std::string& unique_name,
     block_size = config.get<size_t>(unique_name, "block_size");
     samples_per_data_set = config.get<size_t>(unique_name, "samples_per_data_set");
     max_age = config.get_default<float>(unique_name, "max_age", 60.0);
+    fpga_dataset = config.get_default<dset_id_t>(unique_name, "fpga_dataset", dset_id_t::null);
```

Comment:
The dataset ID is in the config under `fpga_dataset/id`, so I think this should be

```suggestion
    fpga_dataset = config.get_default<dset_id_t>("fpga_dataset", "id", dset_id_t::null);
```

To test if this line works, we could add
```
"fpga_dataset": {
      "id": "d3c527df45f258dbd88c6ea7f56eadba"
}
```
to `accumulate_params` in `tests/test_accumulate.py`

Reply:
This is from `coco status`:
```
cluster:
...
  fpga-dataset:
    id: d3c527df45f258dbd88c6ea7f56eadba
```